Ensure that the sourcemap is automatically copied alongside the production bundle

**Library Affected**
webpack workbox plugin 

**Browser & Platform**:
Google Chrome v58

**Issue or Feature Request Description**:
DevTools failed to parse SourceMap: http://localhost/workbox-sw.prod.v1.0.1.js.map

Source map file not generated.

This is pretty much how I'm building. I stripped off some workbox flags that it seemed like I didn't need (but maybe I do), so it could be pilot error on my part.

```
const gulp = require('gulp'),
    child = require('child_process'),
    gutil = require('gulp-util'),
    wb = require('workbox-build');

function buildSw() {
  return wb.generateSW({
    globDirectory: './_site/',
    swDest: './_site/sw.js',
    staticFileGlobs: ['**\/*'],
  }).then(() => {
    gutil.log('Service worker generated.');
  }).catch((err) => {
    gutil.log('[ERROR] This happened: ' + err);
  });
}

gulp.task('build', () => {
  const jekyll = child.spawn('bundle',
      ['exec', 'jekyll', 'build']);
  const logger = (buffer) => {
    buffer.toString()
        .split(/\n/)
        .forEach((message) => gutil.log(message));
  };
  jekyll.stdout.on('data', logger);
  jekyll.stderr.on('data', logger);
  buildSw();
});
```

--- a/packages/workbox-build/src/lib/utils/copy-workbox-sw.js
+++ b/packages/workbox-build/src/lib/utils/copy-workbox-sw.js
@@ -8,7 +8,10 @@ module.exports = (outputDirectory) => {
   const workboxSWSrcPath = require.resolve('workbox-sw');
   const workboxSWDestPath = path.join(outputDirectory,
     path.basename(workboxSWSrcPath));
-  return fse.copy(workboxSWSrcPath, workboxSWDestPath)
+  return Promise.all([
+      fse.copy(workboxSWSrcPath, workboxSWDestPath),
+      fse.copy(`${workboxSWSrcPath}.map`, `${workboxSWDestPath}.map`),
+    ])
     .then(() => workboxSWDestPath)
     .catch((error) => {
       throw Error(`${errors['unable-to-copy-workbox-sw']} ${error}`);

--- a/packages/workbox-build/test/node/utils-copy-sw-lib.js
+++ b/packages/workbox-build/test/node/utils-copy-sw-lib.js
@@ -3,6 +3,7 @@ const path = require('path');
 const errors = require('../../src/lib/errors.js');
 
 require('chai').should();
+const expect = require('chai').expect;
 
 describe('Copy SW Lib', function() {
   it('should reject with an error when the copy fails', function() {
@@ -28,9 +29,27 @@ describe('Copy SW Lib', function() {
   it('should resolve with the file name after the copy completes', function() {
     this.timeout(5 * 1000);
 
+    let jsFileCopied = false;
+    let mapFileCopied = false;
+
     const copyWorkboxSW = proxyquire('../../src/lib/utils/copy-workbox-sw', {
       'fs-extra': {
-        copy: () => Promise.resolve(),
+        copy: (inputFilePath, outputFilePath) => {
+          // Make sure the filenames are the same from input and output paths.
+          expect(path.basename(inputFilePath)).to.equal(path.basename(outputFilePath));
+          switch (path.extname(inputFilePath)) {
+            case '.js':
+              jsFileCopied = true;
+              break;
+            case '.map':
+              mapFileCopied = true;
+              break;
+            default:
+              throw new Error('Unexpected file extension: ' + inputFilePath);
+          }
+
+          return Promise.resolve();
+        },
       },
     });
 
@@ -49,6 +68,9 @@ describe('Copy SW Lib', function() {
         throw new Error('Unexpected result from copying workboxSW: ' +
           workboxSWPath);
       }
+
+      expect(jsFileCopied).to.equal(true);
+      expect(mapFileCopied).to.equal(true);
     });
   });
 });

